lib/handler/status/durations.c

```diff
@@ -64,6 +69,11 @@ static void durations_status_per_thread(void *priv, h2o_context_t *ctx)
         ADD_DURATION(response_time);
         ADD_DURATION(total_time);
 #undef ADD_DURATION
+        if (agg_stats->stats.evloop_latency_ms.capacity <= ctx->thread_index) {
+            h2o_vector_reserve(NULL, &agg_stats->stats.evloop_latency_ms, ctx->thread_index + 1);
+        }
+        agg_stats->stats.evloop_latency_ms.entries[ctx->thread_index] = h2o_evloop_get_execution_time(ctx->loop);
+        agg_stats->stats.evloop_latency_ms.size++;
```

Comment:
... and then `agg_stats->stats.evloop_latency_ms.size = ctx->thread_index + 1` in the `if` above